account_invoice_tax_note/reports/report_invoice_document.xml

```diff
@@ -0,0 +1,20 @@
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html) -->
+<odoo>
+    <template
+        id="report_invoice_document_inherit"
+        inherit_id="account.report_invoice_document"
+    >
+        <span t-field="o.fiscal_position_id.note" position="after">
+            <t
+                t-set="account_tax_groups"
+                t-value="o._get_account_tax_groups_with_notes()"
+            />
+            <t t-if="account_tax_groups">
+                <t t-foreach="account_tax_groups" t-as="account_tax_group">
+                    <p t-field="account_tax_group.report_note" />
+                </t>
+            </t>
+        </span>
```

Comment:
This line should be changed to the following:

```
</p>
```

Reply:
Hi @JavierIniesta I've done the changes as per your feedback. Kindly review it.